Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -345,7 +345,7 @@ void NetworkStorageManager::estimate(const WebCore::ClientOrigin& origin, Comple
     completionHandler(originStorageManager(origin).estimate());
 }
 
-void NetworkStorageManager::clearStorageForTesting(CompletionHandler<void()>&& completionHandler)
+void NetworkStorageManager::resetStoragePersistedState(CompletionHandler<void()>&& completionHandler)
```

Comment:
Does this function guarantee removing all data from disk? At first glance it seems to change the storage mode from Persistent to BestEffort. I'm not sure whether BestEffort guarantees deletion.

Reply:
As in the commit message: "Rename clearStorage() to resetStoragePersistedState() to reflect its
actual use, and introduce a new clearStorage() function to clear multiple types of data at a time.", the naming is confusing.

Current clearStorageForTesting only reset persisted state.